DataSetExplorer/AnnotationConsistency/AnnotationConsistencyService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using DataSetExplorer.AnnotationConsistencyTests;
+using FluentResults;
+using System.Collections.Generic;
+
+namespace DataSetExplorer
+{
+    class AnnotationConsistencyService : IAnnotationConsistencyCheckerService
+    {
+        private FullDataSetFactory _fullDataSetFactory;
+
+        public AnnotationConsistencyService(FullDataSetFactory fullDataSetFactory)
+        {
+            _fullDataSetFactory = fullDataSetFactory;
+        }
+
+        public Result<Dictionary<string, Dictionary<string, string>>> CheckMetricsSignificanceBetweenAnnotatorsForSeverity(int severity)
+        {
+            var instancesGroupedBySmells = _fullDataSetFactory.GetAnnotatedInstancesGroupedBySmells(annotatorId: null);
+            IMetricsSignificanceTester tester = new AnovaTest();
+            return tester.TestBetweenAnnotators(severity, instancesGroupedBySmells);
```

Comment:
For now, I would decoupled the Result class from our Domain Model. Currently, your (M)AnovaTest class returns a Result object, and these are domain services. We can have the application service wrap these results in the Result.Ok for now.